events/models.py

```diff
@@ -27,10 +28,17 @@ class Event(models.Model):
     start_date = models.DateTimeField()
     finish_date = models.DateTimeField()
     venue = models.ForeignKey(Venue, on_delete=models.PROTECT)
+    external_id = models.IntegerField(null=True, blank=True)
 
     def __str__(self):
         return f'{self.name}, {self.start_date:%d.%m.%Y}'
 
+    def save(self, force_insert=False, force_update=False, using=None,
```

Comment:
`*args, **kwargs` for future capability.

Reply:
Done